README.md

```diff
@@ -13,103 +13,37 @@ are required.
   * tcp: `postgres://user:password@site:port/dbname`
   * unix domain socket: e.g. `postgres:///dbname?host=/var/run/postgresql`
   * parameters: e.g. `postgres://localhost/dbname?client_encoding=UTF8`
-* `POSTGRESQL_TABLE`: name of table for storing CouchDB data.
-* `POSTGRESQL_COLUMN`: name of the `jsonb` column in `POSTGRESQL_TABLE` for
-  storing CouchDB data.
-* `COUCHDB_URL`: a full path URL to `_all_docs`, including `user:pass@`, the
-  database name, and `_all_docs`.
-  * e.g. `https://user:pass@localhost/medic/_all_docs`
+* `COUCHDB_URL`: a full path URL to the couchdb database, with any required credentials.
+  * e.g. `https://user:pass@localhost/medic`
 * `COUCH2PG_SLEEP_MINS`: number of minutes between checking for updates.
 
 Optional variables:
 
-* `COUCH2PG_DOC_LIMIT`: maximum number of full documents to request and download from couch during any particular iterative run. this is useful to avoid out of memory errors. Must be balanced properly with `COUCH2PG_SLEEP_MINS` to keep up with new data but not overload.
-* `COUCH2PG_DEBUG`: set this to anything to get more output.
+* `COUCH2PG_DOC_LIMIT`: number of documents to grab concurrently. Defaults to 100. Increasing this number will cut down on HTTP GETs and may improve performance, decreasing this number will cut down on node memory usage, and may increase stability.
+* `COUCH2PG_DEBUG`: returns more debug output from the xmlforms module.
 
 ## Required database setup
 
-It is assumed PostgreSQL is in use.
+We support PostgreSQL 9.4 and greater. The user passed in the postgres url needs to have full creation rights on the given database.
 
-### `full_access` role
+## Running tests
 
-A role called `full_access` must exist, and the role accessing PostgreSQL must
```

Comment:
Yeah me neither. Apart from "in the code" I don't know about any Medic documentation. I presume this is something a Tech Lead would know about? Regardless, I agree with you that it should be somewhere in our internal documentation and not our open source space.